# Hypothesentest - Mittelwert
hier die zugrundeliegende [Theorie](https://github.com/JHC90/Basic-DataScience-Skills/wiki/Statistics_Basics_020Hypothesentests)

## Aufgabenstellung
Ein Unternehmen möchte seine Website-Leistung verbessern. <br>
Derzeit haben Seiten eine mittlere Ladezeit von 3,125 Sekunden mit einer Standardabweichung von 0,700 Sekunden. Sie beauftragen ein Beratungsunternehmen, um die Ladezeiten zu verbessern. <br>
Das Management möchte ein Konfidenzniveau von 99% haben. <br>
Ein Beispiellauf von 40 neuen Seiten hat eine mittlere Ladezeit von 2,875 Sekunden.<br>
Sind diese Ergebnisse statistisch schneller als zuvor?

## Lösung
<b>Schritt1: Informationen aus der Angabe</b><br>
𝜇=3.125<br>
𝜎=0.700<br>
𝛼=0.01<br>
𝑛=40<br>
𝑥 ̅=2.875<br>



<b>Schritt2: Hypothesen aufstellen</b><br>
1. Nullhypothese: 𝐻_0: 𝜇≥3.125
2. Alternativhypothese: 𝐻_1: 𝜇<3.125
3. Das Signifikanzniveau: 𝛼=0.01
4. Ermittle den testtyp: Left tail  (weil h1 <mark>"kleiner <"</mark> 3.125)

![title](https://raw.githubusercontent.com/wiki/JHC90/Basic-DataScience-Skills/imgs/h1Smaller0.PNG)
mit 𝛼=0.01

### Traditionelle Methode
$Z=\frac{\bar{x}-\mu}{\sigma / \sqrt{n}}=\frac{2.875-3.125}{0.7 / \sqrt{40}}=-2.259$ || Z(= großes Z, das wir errechnen)

In [21]:
Z = (2.875-3.125) / (0.7/(40**0.5))
Z

-2.2587697572631282

Prüfe diesen Wert gegen den, den ich aus der z-Tabelle ablese
![title](https://raw.githubusercontent.com/wiki/JHC90/Basic-DataScience-Skills/imgs/h1Smaller0MitZ.PNG)
mit 𝛼=0.01 ist z dann "-2.325"
![title](./imgs/AufgabeHypothesentest2.PNG)

hier ist ein Top sonderfal...der konkrete Wert von 𝛼=0.01 exisitiert nicht in der Tabelle, in diesem Fall nehme ich die beiden werte links und rechts und setze den wert dazwischen (man könnte hier auch dritteln, jeodch ist in der Muster lösung der Wertz halbiert => an das halte ich mich somit auch hier weiterhin) ... Sprich (-2.32 + (-2.33))/2 = -2.325
mit 𝛼=0.01 ist z dann 𝛼=0.01 ist dann 

Der krtitsche Wert liegt somit bei -2.325 = z(kleines z) // großes Z wurde bereits berrechnet = -2.258

Wenn Z>z => kann Nullhypothes angenommen werden => Nullhypothese besagt "Nullhypothese: 𝐻_0: 𝜇≥3.125" = sprich die neuen Webseiten sind nicht statistisch schneller, zumindest nicht nach dem traditionellen Verfahren.


<b>Ergebnis Traditionelle Methode</b>
Die Nullhypothese kann angenommen werden, da => man lehnt den Vorschlag ab, da laut dieser Messung, das Ergebnis nicht gut genug ist bzw "Wir können nicht sagen, dass die neuen Webseiten statistisch schneller sind.

"Bei  "−2,259>−2,325", " liegt die Teststatistikaußerhalb des Ablehungsbereichs

Wir können nicht sagen, dass die neuen Webseiten statistisch schneller sind.

### P-Test

Greife zurück auf berechnetes (großes Z).<br>
Z = -2.259

Check die Z-Tabelle bei -2.26
![title](./imgs/AufgabeHypothesentest3.PNG)

Den abgelesen Wert checke ich nun gegen mein gegebenes Alpha.


abgelesen Wert > Alpha => Nullhypothese annhemen<br>
abgelesen Wert < Alpha => Nullhypothese verwerfen

<b>Ergebnis P-Wert Methode</b><br>
Die Nullhypothese kann angenommen werden<br>
"Bei  " 0,0119>0,01", " liegt der P−Wertüber dem Signifikanzniveau 𝛼


<b>Wort zum Sonntag</b><br>
Laut dieser Tests kann der Ergebnisvorschlag verworfen werden. In diesem Fall würde man die Consultants bitten nochmals nachzubessern